Documentation/getting-started-guide.md

```diff
@@ -22,6 +22,41 @@ func main() {
 }
 ```
 
+### Use systemd notification mechanism
```

Comment:
I'm not sure this belongs in the getting-started-guide - it's a bit overwhelming for first-time users. We can probably let this go, but it would be good to clean up.